packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -33,8 +33,6 @@ export function cloudflare<T extends Record<string, WorkerOptions>>(
 			return {
 				resolve: {
 					alias: getNodeCompatAliases(),
-					// We want to use `workerd` package exports if available (e.g. for postgres).
-					conditions: ['workerd'],
```

Comment:
@petebacondarwin I needed to move this to the environment config since `conditions` is no longer inherited by environments ([source change](https://github.com/vitejs/vite/commit/d002e7d05a0f23110f9185b39222819bcdfffc16#diff-11e17761d4ecfee8f8fde15c6d79b7bc0260176396a30dfd8e6f6bbaf5af4745R682)) I hope this is fine, was there a particular reason why you added the `conditions` here?

Reply:
This was the place I found that worked. But if it works in the environment config that is also cool. 
Does that cover builds too?

Reply:
> Does that cover builds too?

It is a top-level configuration so I would definitely say yes 🤔